android/src/main/java/com/lwansbrough/RCTCamera/MutableImage.java

```diff
@@ -167,7 +167,7 @@ private static Bitmap toBitmap(byte[] data) {
     }
 
     public String toBase64(int jpegQualityPercent) {
-        return Base64.encodeToString(toJpeg(currentRepresentation, jpegQualityPercent), Base64.NO_WRAP);
+        return Base64.encodeToString(toJpeg(currentRepresentation, jpegQualityPercent), Base64.DEFAULT);
```

Comment:
is this a breaking change?

Reply:
Sorry, I just see I have a conflict, with master branch. Will fix and commit.